### PR TITLE
Fix `release-download-count` spacing

### LINKED ISSUE #6150: `release-download-count` missing after 100 results
### Description

Firefox_Screenshot_2022-11-09T17-44-59 229Z

### How to replicate the issue + URL

https://github.com/veracrypt/VeraCrypt/releases/tag/VeraCrypt_1.25.9

### Extension version

22.11.6

### Browser(s) used

FireFox 106.0.5 64-bit

### LINKED ISSUE #6509: `release-download-count` lacks spacing on small screen
### Description

image

### How to replicate the issue + URL

https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.8.4

### Extension version

23.4.10

### Browser(s) used

Chrome

### PR DESCRIPTION
**Pull request by kidonng**
Fix #6509
Fix #6150

## Test URLs

https://github.com/veracrypt/VeraCrypt/releases/tag/VeraCrypt_1.25.9

## Screenshot

<table>
<tr>
	<td>Before
	<td>After
<tr>
	<td>image
	<td>

[See comment](https://github.com/refined-github/refined-github/pull/6510#issuecomment-1508788214)

</table>

### CODE REVIEW

**Comment by fregante** [comment]
I'd probably increase the spacing, it doesn't seem enough here. Maybe use a wider margin just on mobile

Also an annoying GitHub bug is the box-icon spacing, if you feel like fixing that

### Native

Screenshot 2


### `margin-right: 0.4em;`

Screenshot 1

**Comment by kidonng** [comment]
https://user-images.githubusercontent.com/44045911/232056844-a67a1110-1b4c-4f22-9408-c18de824b49c.mov

File: source/features/release-download-count.css [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+	min-width: 6em;
 }
 
-/* Simulates fixed columns: Size  */
-.rgh-release-download-count .text-sm-left {
-	min-width: 5em;
-}
-/* Simulates fixed columns: Date */
-.rgh-release-download-count .text-right {
-	min-width: 6em;
```
Comment (fregante) [inline source/features/release-download-count.css:12]: Make sure this cause a regression (you can blame it to find it)

**Review by fregante — APPROVED** [review]
Looks great. Only one check before merging 👇

File: source/features/release-download-count.css (hunk 1 above)
Comment (kidonng) [reply source/features/release-download-count.css:12]: This is included above (`.rgh-release-download-count > :is(span, small)` also selects this)

Comment (fregante) [reply source/features/release-download-count.css:12]: Ah all good, I was referring to this one, but we haven't fixed it yet:

- https://github.com/refined-github/refined-github/issues/6150

Does this PR fix it by chance?

Comment (kidonng) [reply source/features/release-download-count.css:12]: Nope, should we fix it here or in a new PR?

Comment (fregante) [reply source/features/release-download-count.css:12]: Here would be ok since this PR is about alignment too, if you'd like

**Comment by kidonng** [comment]
Fixed for #6150

image

image

**Comment by kidonng** [comment]
😅 oops, need to find if there's an alternative to center-aligning the asset size

**Comment by kidonng** [comment]
All set! I was fighting `flex-grow` all this time when I should change `flex-basis` 🥲

image

image

**Review by fregante — APPROVED** [review]
Looks good. I'd be open to making the downloads count text non-small too for consistency

File: source/features/github-bugs.css [hunk 2]
```diff
@@ -81,3 +81,8 @@
 		display: none;
 	}
 }
+
+/* Add margin to release download icon #6510 */
+:is(.octicon-package, .octicon-file-zip):has(+ a[rel='nofollow']) {
```
[the remark this answers is not in the document]
Comment (kidonng) [reply source/features/github-bugs.css]: > Is there any additional class that can help limit this? Maybe Box?

I prepended `.Box-row`, but honestly the markup is just too generic, you never know

**Pull request merged**